model/gitlab/gitlab.go

```diff
@@ -47,7 +47,8 @@ func userFromGitLabUser(glu *GitLabUser) *model.User {
 	}
 	strings.TrimSpace(user.Email)
 	user.Email = glu.Email
-	*user.AuthData = strconv.FormatInt(glu.Id, 10)
```

Comment:
here user is created with `&model.User{}` and have AuthData which is a pointer on a string.
But the default of a pointer is `nil`, no memory is allocated and is not pointing to an address and this part of the code it tries to put the value from `strconv.FormatInt(glu.Id, 10)` in the address pointed by `AuthData` which is not existing.